integrations/other.rst

```diff
@@ -7,54 +7,55 @@ Custom integrations
 If you intend to use a build system that does not have a built-in generator, you may still be 
 able to do so. There are several options:
 
-- First, search in bintray. Generators can now be created and contributed by users as regular
-  packages, so you can depend on them, use versioning, and evolve faster without depending on the
-  conan releases. See :ref:`generator packages <dyn_generators>`.
-- You can use the **text or json generator**. It will generate a text file, simple to read and to parse
-  that you can easily parse with your tools to extract the required information.
-- Use the **conanfile data model** and access its properties and values, so you can directly
-  call your build system with that information, without requiring to generate a file.
-- Write and **create your own generator**. So you can upload it, version and reuse it, as well
-  as share it with your team or community. Check :ref:`generator packages <dyn_generators>` too.
-  
-  
-.. note:: 
-   
-   Need help integrating your build system? Tell us what you need. info@conan.io
+- First, search in Bintray for generator packages. Generators can be created and contributed by users as regular packages, so you can depend
+  on them as a normal requirement, use versioning and evolve faster without depending on the Conan releases. See :ref:`custom_generator`.
 
+- You can use the :ref:`text_generator` or :ref:``json_generator`` generators. They will generate a text file, simple to read that you can
+  easily parse with your tools to extract the required information.
+
+- Use the **conanfile data model** (ref:`deps_cpp_info_attributes_reference`, :ref:`deps_env_info_attributes_reference`) and access its
+  properties and values, so you can directly call your build system with that information, without requiring to generate a file.
+
+- Write and **create your own generator**. So you can upload it, version and reuse it, as well as share it with your team or community.
+  Check :ref:`dyn_generators`.
```

Comment:
If we move this entry list to the second place, we can remove from the first entry the comment about creating a _custom generator_.